ministryofjustice/laa-apply-for-criminal-legal-aid
# Redirect to dashboard if user is signed in

## Description of change
When a use (provider) is not signed in and they access the service through a link or by entering the URL, they see the home page with the start button.

If they are signed in, they get redirected to the applications dashboard instead. This happens if they click the header link (service name) or if they go to the root URL, etc.

This is a more consistent behaviour than showing again the "Start now" button. Simplified the view and removed the partial.

## Link to relevant ticket

## Notes for reviewer

## How to manually test the feature
Without signing in, access the root of the service. The start now button should show. Now sign in (click the start now button) and you are redirected to the dashboard. If you click the header service name, or access the `/` root of the service you get redirected back to the dashboard.
